dev/WindowsAppRuntime_MSIXInstallFromPath/main.cpp

```diff
@@ -138,6 +139,9 @@ HRESULT JustDoIt(PCWSTR path) noexcept try
     auto inventory{ fpath / L"MSIX.inventory" };
     std::string lineUtf8;
     std::ifstream f{ inventory };
+
+    THROW_HR_MSG(HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND), "%ls", inventory.c_str());
```

Comment:
Looks like a bug. Did you intend to have this always throw?

Reply:
Likely you meant THROW_HR_IF_MSG?

Reply:
aargh!! copy pasted Howard's feedback as is. my bad. thanks for catching it.